cmd/kubeadm/app/phases/upgrade/compute_test.go

```diff
@@ -188,43 +205,45 @@ func TestGetAvailableUpgrades(t *testing.T) {
 		},
 		{ // allow experimental upgrades, but no upgrade available
 			vg: &fakeVersionGetter{
-				clusterVersion: "v1.8.0-alpha.2",
-				kubeletVersion: "v1.7.5",
-				kubeadmVersion: "v1.7.5",
+				clusterVersion: "v1.9.0-alpha.2",
+				kubeletVersion: "v1.8.5",
+				kubeadmVersion: "v1.8.5",
 
-				stablePatchVersion: "v1.7.5",
-				stableVersion:      "v1.7.5",
-				latestVersion:      "v1.8.0-alpha.2",
+				stablePatchVersion: "v1.8.5",
+				stableVersion:      "v1.8.5",
+				latestVersion:      "v1.9.0-alpha.2",
 			},
 			expectedUpgrades:  []Upgrade{},
 			allowExperimental: true,
 			errExpected:       false,
 		},
 		{ // upgrade to an unstable version should be supported
 			vg: &fakeVersionGetter{
-				clusterVersion: "v1.7.5",
-				kubeletVersion: "v1.7.5",
-				kubeadmVersion: "v1.7.5",
+				clusterVersion: "v1.8.5",
+				kubeletVersion: "v1.8.5",
+				kubeadmVersion: "v1.8.5",
 
-				stablePatchVersion: "v1.7.5",
-				stableVersion:      "v1.7.5",
-				latestVersion:      "v1.8.0-alpha.2",
+				stablePatchVersion: "v1.8.5",
+				stableVersion:      "v1.8.5",
+				latestVersion:      "v1.9.0-alpha.2",
 			},
 			expectedUpgrades: []Upgrade{
 				{
 					Description: "experimental version",
 					Before: ClusterState{
-						KubeVersion: "v1.7.5",
+						KubeVersion: "v1.8.5",
 						KubeletVersions: map[string]uint16{
-							"v1.7.5": 1,
+							"v1.8.5": 1,
 						},
-						KubeadmVersion: "v1.7.5",
+						KubeadmVersion: "v1.8.5",
 						DNSVersion:     "1.14.5",
+						EtcdVersion:    constants.MinExternalEtcdVersion,
```

Comment:
minor nit: I think I prefer to code the strings here as-is so it's easy to see the versions being used

Reply:
done